### DISCUSSION TITLE
Picture-wall is empty  Cause: cache not writable

### DISCUSSION THREAD

**Discussion by DarkDau** [shaarli/Shaarli#2007] [Show and tell]
I get "There is no cached thumbnail / thumbnail. Try to synchronize them."
It was working perfectly until today. I did change the theme to material yesterday. This evening is no picture-wall tried back and fourth again, sync the thumbnails several time. deleting the browser cache, does not help either.
So far so bad.
I found it by myself: The cache directory was set to the wrong owner and therefor it was **not writeable**.

To bad when the access permissions for the cache directory were not set correctly and the routine in the backend still showed the permissions as correct. 

This should in need of improvement. It would be good to include a check for this in the next update.

**Comment by nodiscc** [comment]
> the routine in the backend still showed the permissions as correct.

I cannot reproduce this. Whenever I change the permissions of `/cache` to simulate a non-writable cache directory, the permissions check at `/admin/server` correctly reports the problem:

```bash
$ ls -ld /var/www/links.example.org/cache/ # correct
drwxrwx--- 4 www-data shaarli 4096 Jul 12 21:25 /var/www/links.example.org/cache/
$ sudo chmod g-w /var/www/links.example.org/cache/ # make it incorrect on purpose
```

![image](https://github.com/shaarli/Shaarli/assets/3950390/4e4cc83d-38a8-45a9-a63a-a0bfd91c9e98)

So I don't think there is anything to fix here, unless I missed something.

**Reply by DarkDau** [reply]
strange, but ok.
I had wrong user as well. group were ok.
Ok nevermind, sorry for complaining
